package.json

```diff
@@ -24,13 +24,15 @@
     "@types/react": "^17.0.33",
     "@types/react-dom": "^17.0.10",
     "@types/webextension-polyfill": "^0.8.1",
+    "bootstrap": "^5.1.3",

```

Comment:
See discord, but double check if you need both bootstrap and react-bootstrap. If you do, keep as is. If not, remove the unneeded one. Also ping the chat to let everyone know they need to `npm install` again.